.github/workflows/build.yml

```diff
@@ -19,7 +19,7 @@ on:
 
 jobs:
   prepare:
-    runs-on: ubuntu-20.04
```

Comment:
what do you think if use string/constant/global var :) for this? in future, for example, change only 1 var instead of every job:)

Reply:
I also thought about that, but iirc GitHub Actions wants a static value to be manually defined or else I'd expect it to be supported in the `defaults` keyword.  

There might be some ways to work around this, but I wouldn't block merging because of it :)